x-pack/plugins/reporting/server/lib/store/mapping.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
```

Comment:
This is starting to feel more like a "constant" to me since it's mostly static. Not sure on your thoughts there

Reply:
We may need to visit this more often than other constants. I think it's best to keep this easy to find.